app/controllers/assets_controller.rb

```diff
@@ -38,8 +38,9 @@ def restrict_request_format
   end
 
   def asset_params
+    asset_params = handle_empty_access_limited_param(params)
```

Comment:
I think it would be useful to explain why you have to operate on the `params` *before* the calls to `require` and `permit`.

Reply:
I've updated the commit note to explain why I need to modify the params before calling `require` and `permit`.